packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -212,6 +212,17 @@ export class EditorToolbarComponent {
 
 	/* Publish and unpublish */
 
+	/**
+	 * Returns the text present for the save/publish button.
+	 *
+	 * @returns {Promise<string>} String found on the button.
+	 */
+	async getPublishButtonText(): Promise< string > {
```

Comment:
This method is part of the refactor to the `publish` method to eliminate relying on flaky UI elements.